### PR TITLE
Add missing neox rpc client wrappers

### PR DESCRIPTION
**Pull request by fyrchik**
Related #1012 .

### CODE REVIEW

File: pkg/rpc/client/state.go [hunk 1]
```diff
@@ -48,3 +48,14 @@ func (c *Client) GetProof(root util.Uint256, sc util.Uint160, key []byte) (*resu
 	}
 	return &resp, nil
 }
+
+// VerifyProof verifies keyed proof for the state with the specified root.
+func (c *Client) VerifyProof(root util.Uint256, proof *result.ProofWithKey) (*result.VerifyProof, error) {
```
Comment (roman-khimov) [inline pkg/rpc/client/state.go:53]: Any tests for this?

File: pkg/rpc/client/state.go [hunk 2]
```diff
@@ -35,3 +37,14 @@ func (c *Client) GetStateHeight() (*result.StateHeight, error) {
 	}
 	return resp, nil
 }
+
+// GetProof returns proof that key belongs to a contract h state rooted at root.
```
Comment (roman-khimov) [inline pkg/rpc/client/state.go:41]: `contract sc state` probably.

Comment (fyrchik) [reply pkg/rpc/client/state.go:41]: FIxed.

File: pkg/rpc/client/state.go (hunk 1 above)
Comment (fyrchik) [reply pkg/rpc/client/state.go:53]: Fixed.

**Pull request merged**